plover/command/set_config.py

```diff
@@ -0,0 +1,41 @@
+import ast
+
+COMMAND_DELIMITER = ','
+
+
+def plugin_error(e, msg):
+    """ Add an additional message to an exception before raising to the engine. """
+    e.args = (e.args[0] + "\nSET_CONFIG: " + msg,)
```

Comment:
if you PEP8 this file I think it'll catch a few things. Can't remember if in python, single-quotes are preferred